src/Chainweb/Version.hs

```diff
@@ -783,7 +783,7 @@ twentyChainUpgrade
     -> Bool
 twentyChainUpgrade Mainnet01 _ h = h == maxBound -- TODO: update me with a calculated height
 twentyChainUpgrade Development _ h = h == 150
-twentyChainUpgrade _ _ 2 = True
+twentyChainUpgrade _ _ 3 = True
```

Comment:
Only bumps for testing versions. Adding remediations to the test chainweb versions broke alot of our tests, which assume invariants about the quantity of txs and their block heights. `coinV2Upgrade` happening at 1 broke those invariants, and bumping it to block height 2 is safe, and fixes those problems.